components/_global-navigation.scss

```diff
@@ -47,25 +55,101 @@ $nav-height: 49px;
 	}
 
 	&__middle-column {
+		background-color: $nav-background-color;
+		display: flex;
+	}
+
+	&__notifications {
+		position: relative;
+	}
+
+	&__notifications-counter-wrapper {
+		align-items: center;
+		background-color: $color-link;
+		border: 1px solid $color-white;
+		border-radius: $notification-counter-size / 2;
+		box-sizing: border-box;
+		color: $color-white;
+		display: flex;
+		font-size: $typescale-size-minus-3;
+		height: $notification-counter-size;
+		justify-content: center;
+		left: 11px;
+		min-width: $notification-counter-size;
+		position: absolute;
+		top: 7px;
+
+		@media #{$breakpoint-desktop-xl-up} {
+			left: 36px;
+		}
+	}
+
+	&__notifications-counter {
+		padding: 0 4px;
+	}
+
+	&__dropdown-toggle {
+		align-items: center;
 		display: flex;
+		justify-content: center;
+	}
+
+	&__dropdown {
+		color: $color-dark-blue-gray;
+		width: 190px;
+		z-index: -1;
+
+		.wds-button {
```

Comment:
Should we use inheritance here and style based on context? Or should we create separate classes like `wds-global-navigation__account-navigation-button`? Or maybe add a modifier to the base definition of the (in this case) button?